draft-ietf-lake-edhoc.md

```diff
@@ -1090,9 +1094,11 @@ Implementations MAY support message_4. Error codes 1 and 2 MUST be supported.
 
 Implementations MAY support EAD.
 
+Padding is OPTIONAL to support.
```

Comment:
Perhaps you mean "Including padding to an EDHOC message is OPTIONAL to support."

That is, I would expect the active use of padding from a sender node to be indeed optional.

However, a recipient node always has to be able to parse an EDHOC message, also in case padding is included. This is already what seems to be intended above in this PR, about the updated message format and processing.

Reply:
"That is, I would expect the active use of padding from a sender node to be indeed optional."
Yes, I think that is already in the PR.

"However, a recipient node always has to be able to parse an EDHOC message, also in case padding is included."
Not necerasy as discussed by Carsten during the last COSE meeting. You could live when the
recipient failing nicely. CBOR makes this possible. The recipient would know just now that does not expect
true in the beginning of the plaintext and fail. It does not need to know the meaning of the true. But
mandatory to support padding removal for recipient might be good.